pom.xml

```diff
@@ -85,13 +85,13 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
-      <version>2.8.10</version>
+      <version>2.8.11</version>
```

Comment:
I think you will get a `requireUpperBoundDeps` error here and will need to explicitly depend on `jackson-core:2.8.11`, since `jackson-datatype-jsr310` depends on 2.8.11 but `jackson-databind`depends on 2.8.10.